### DISCUSSION TITLE
[BUG] Rust Server Crashing

### DISCUSSION THREAD

**Discussion by LucianDev** [GameServerManagers/LinuxGSM#3798] [Q&A]
Rust server cashes as soon as while booting up, I've noticed while watching the console looks like it runs a CTRL+C on itself and the console logs don't seem helpful to me can't figure out what exactly the issue is.

## Basic info

* **Distro:** [Ubuntu 20.04]
* **Game:** RUST
* **Command:** START
* **LinuxGSM version:** v21.5.1

## Further Information

Posted the console log here https://pastebin.com/EN5U0WM3 due to how long it is.

## Expected behavior
To start the server fully without closing

**Comment by johnoclockdk** [comment]
i dont think that the case  here try to reinstall rust / deps

**Comment by Claiyc** [comment]
Does not seem to be an issue with lgsm. Moving to discussions.
